vignettes/DALEX_teaching2.Rmd

```diff
@@ -1,176 +1,179 @@
-sv_xgb  <- single_variable(explainer_xgb, variable =  "surface", type = "pdp")
-sv_svm_e1071 <- single_variable(explainer_svm_e1071, variable =  "surface", type = "pdp")
-sv_tr <- single_variable(explainer_tr, variable =  "surface", type = "pdp")
-
-plot(sv_lm, sv_rf, sv_svm, sv_svm_e1071, sv_xgb, sv_tr)
-
-```
-
-For the variable `surface` we see that all of the models have nearly the same performance and for this variable we see that the apartments with larger surface have the smaller mean price per square meter, than flats with the smaller surface.
-
-##Floor
-```{r var_floor}
-sv_rf  <- single_variable(explainer_rf, variable =  "floor", type = "pdp")
-sv_lm  <- single_variable(explainer_lm, variable =  "floor", type = "pdp")
-sv_svm  <- single_variable(explainer_svm, variable =  "floor", type = "pdp")
-sv_xgb  <- single_variable(explainer_xgb, variable =  "floor", type = "pdp")
-sv_svm_e1071 <- single_variable(explainer_svm_e1071, variable =  "floor", type = "pdp")
-sv_tr <- single_variable(explainer_tr, variable =  "floor", type = "pdp")
-
-plot(sv_lm, sv_rf, sv_svm, sv_svm_e1071, sv_xgb, sv_tr)
-
-
-```
-
-For the variable `floor` we see that all of the models have nearly the same performance and for this variable we see that the apartments which are on the higher floor have the smaller mean price per square meter, than flats on the lower floors. As in the case with the variable `surface` we see that the svm model gives us the smallest mean price than other models.
-
-
-##District
-```{r var_district, fig.height=15}
-sv_rf  <- single_variable(explainer_rf, variable =  "district", type = "factor")
-sv_lm  <- single_variable(explainer_lm, variable =  "district", type = "factor")
-sv_svm  <- single_variable(explainer_svm, variable =  "district", type = "factor")
-sv_svm_e1071 <- single_variable(explainer_svm_e1071, variable =  "district", type = "factor")
-sv_tr <- single_variable(explainer_tr, variable =  "district", type = "factor")
-
-
-plot(sv_lm, sv_rf, sv_svm, sv_svm_e1071, sv_tr)
-
-```
-
-For the variable `district` we see that all of the models have nearly the same performance. We see on the plots three clusters: the city center (Srodmiescie), districts well communicated with city center (Ochota, Mokotow, Zoliborz) and other districts closer to city boundaries.
-
-#Session info
-```{r session info}
-devtools::session_info()
+---
+title: "What they have learned? Part 2"
+author: 
+  - Aleksandra Grudziąż, Anna Kozak
+date: First created on May 29, 2018. Updated on Nov 09, 2020
+output: 
+  html_document:
+    toc: true  
+    toc_float: true
+    number_sections: true
+    theme: flatly
+    highlight: kate
+    fig_caption: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE, message=FALSE, warning=FALSE)
+```
+
+In the vignette below we will see how four selected models of predicting the price of flats find relationships between dependent variables and variables from the model.

```

Comment:
how a four selected models that predict the price of flats find relationships ...

Reply:
Fixed.